virttest/utils_net.py

```diff
@@ -1894,7 +1894,10 @@ def ovs_br_exists(brname, ovs=None):
     if ovs is None:
         ovs = __ovs
 
-    return brname in ovs.list_br()
+    if ovs is not None:
+        return brname in ovs.list_br()
+    else:
+        raise exceptions.TestError("Host does not support OpenVSwitch")
```

Comment:
But the root cause is we got a None OVS instance here but not BRNotExist? I think maybe "Host does not support OpenVSwitch" is more accurate for debugging.

Reply:
Also, I checked iface_ovs.py in libvirt test case which uses this API and @xutian review comment seems to make sense, because the testcase checks for ovs_br_exists() and decide to configure ovs.

Reply:
I think they are not the same situation. This patch is to deal with ovs not support issues, but not br not exist. As if __ovs is None, list_br will raise AttributeError. I think the change will not affect the iface_ovs test